src/ctrlc/windows.rs

```diff
@@ -2,8 +2,15 @@
 use crate::ctrlc::interrupted::set_interrupted;
 use tracing::error;
 use winapi::shared::minwindef::{BOOL, DWORD, FALSE, TRUE};
-use winapi::um::consoleapi::SetConsoleCtrlHandler;
-use winapi::um::wincon::CTRL_C_EVENT;
+
+// Define only what's needed so we don't require winapi's `consoleapi`/`wincon` features
+// CTRL_C_EVENT value from WinAPI headers
+const CTRL_C_EVENT: DWORD = 0;
+
+extern "system" {
+    // BOOL SetConsoleCtrlHandler(PHANDLER_ROUTINE HandlerRoutine, BOOL Add);
+    fn SetConsoleCtrlHandler(HandlerRoutine: Option<extern "system" fn(DWORD) -> BOOL>, Add: BOOL) -> BOOL;
+}
```

Comment:
Please keep the PR to bumping the `tracing-subscriber` version...
If builds are failing locally, please open another issue or PR for that. If CI is happy I'm happy.